_gsocproposals/proposal_ROOTmpi.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Message Passing Interface for ROOT (ROOTMpi)
+layout: plain
+project: ROOT
+organization: UdeA - CERN
```

Comment:
Would need to change to CERN only. That field is only used to create the list on http://hepsoftwarefoundation.org/gsoc/cern.html 
If you like you can add a line to the very text mentioning which universities participate.